lib/Invoice.php

```diff
@@ -299,8 +299,9 @@ public function addLines($params = null, $opts = null)
      * @param null|array $params
      * @param null|array|string $opts
      *
-     * @return Invoice the created invoice
      * @return Invoice the attached invoice
+     *
+     * @throws Exception\ApiErrorException if the request fails
```

Comment:
@prathmesh-stripe This should have come in via your merge PR - not sure why it got missed